exp/controllers/awsmanagedmachinepool_controller.go

```diff
@@ -18,6 +18,7 @@ package controllers
 
 import (
 	"context"
+	"fmt"
```

Comment:
Which overall cases did you test? Did you include EKS node group testing and are you using it? Asking because I'd like to know which of the cluster types was covered in tests (EKS or non-EKS or both).